src/component/userJoin/Join.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import scrollLogo from "../../image/scrollLogo.png";
+// import axios from "axios";
+import "./join.css";
+
+let user;
```

Comment:
Exclude this user and adhere to the instructions we've gone over.

Reply:
Done this request.